modules/http4s-cloud-functions/src/main/scala/com/github/ingarabr/http4s/Http4sCloudFunctionIOApp.scala

```diff
@@ -1,14 +1,22 @@
 package com.github.ingarabr.http4s
 
-import cats.effect.{Blocker, ContextShift, IO}
+import cats.effect.unsafe.{IORuntime, IORuntimeConfig}
+import cats.effect.IO
 import com.google.cloud.functions.{HttpFunction, HttpRequest, HttpResponse}
 
 trait Http4sCloudFunctionIOApp extends Http4sCloudFunction[IO] with HttpFunction {
 
   val defaultChunkSize = 1024
 
-  implicit val contextShift: ContextShift[IO] = IO.contextShift(CatsExecutionContexts.compute)
-  val blocker: Blocker = CatsExecutionContexts.blocker
+  val (scheduler, _) = IORuntime.createDefaultScheduler()
+
+  implicit val runtime: IORuntime = IORuntime(
+    CatsExecutionContexts.compute,
+    CatsExecutionContexts.blocker,
+    scheduler,
+    () => (),
+    IORuntimeConfig()
+  )
```

Comment:
The `CatsExecutionContext` was more a workaround because cats-effect didn't expose the execution contexts. I think we can remove it in favor of the new global `IORuntime`.

```
implicit val runtime: IORuntime = IORuntime.global
```